lib/database_rewinder/cleaner.rb

```diff
@@ -75,7 +75,7 @@ def reset
     end
 
     def borrow_connection(pool)
-      if ActiveRecord::VERSION::STRING >= '7.2'
+      if ActiveRecord.gem_version >= '7.2'
```

Comment:
https://github.com/rails/rails/blob/v7.2.1/activerecord/lib/active_record/gem_version.rb

Reply:
Thank you! Sorry it took me so long to reply, somehow I missed the notification 🙇

Reply:
@kucho thanks!